AtlasStarter/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using MongoDB.Driver;
+
+namespace AtlasStarter
+{
+    class MainClassNoPrompt
+    {
+        public static void Main(string[] args)
+        {
+            // TODO:
+            // Replace the placeholder connection string below with your
+            // Altas cluster specifics. Be sure it includes
+            // a valid username and password! Note that in a production environment,
+            // you do not want to store your password in plain-text here.
```

Comment:
Would it be a good idea to give directions for using a x.509 certificate here, too?

Reply:
@johnmealy23, good question. I guess it gets back to how simple we want to keep it. I could add a link to the docs here in the comments. Thoughts?

Reply:
I think we should avoid x509 because it will add a lot of complexity, especially for certain drivers that require setup that is platform-specific. Instead, we could link them to other x509 documentation.